src/client/containers/Home/Home.js

```diff
@@ -1,3 +1,4 @@
 import React from 'react';
+import Login from '../LoginPage/LoginPage.component';
 
-export const Home = () => <section className="home">Final project!</section>;
+export const Home = () => <Login />;
```

Comment:
Please remove this line - Home page will not host Login but will instead host basic information.

Reply:
I was told like this: As per wireframes there is no home page, starting page is login page. Is it not true?

Reply:
The starting page can be seen in Figma after the vertical line in each row. Login is a separate page and only available when logging in.